### DISCUSSION TITLE
VideoWriter node did not work

### DISCUSSION THREAD

**Discussion by chongtianyifa** [bonsai-rx/bonsai#1218] [Q&A]
Hi,
I met this weird issue since yesterday. VideoWriter node did not write any video files while it shows the images during running. I changed the file saving path to another disk and it worked for a few hours yesterday. However, it did not work anymore, no matter where the path is.
Any idea how could this happen?

Update: change the suffix of save name from none to filecount or timestamp temporarily works now.
Thanks,
[videowrite.zip](https://github.com/bonsai-rx/bonsai/files/10385601/videowrite.zip)

Lei

**Comment by bruno-f-cruz** [comment]
Can you share the workflow that reproduces the bug? 
Thanks,
Bruno

**Reply by chongtianyifa** [reply]
Just a simple workflow would cause this issue:
![image](https://user-images.githubusercontent.com/46367566/211999982-94c36386-298d-46c9-b9f1-6c2174189607.png)

**Reply by bruno-f-cruz** [reply]
Using that workflow without changing any properties should not even save a video. It should throw an error "A valid path must be specified".

**Reply by chongtianyifa** [reply]
Of course I set the video name with path 'D:\Data\Lei\videos\kineWheel\leftview\testBody.avi'.

**Reply by bruno-f-cruz** [reply]
Does it ever not save the first time you run it then? 
I am just having a hard time helping you out without having a workflow that reproduces the bug. 
sorry =/

**Reply by chongtianyifa** [reply]
I can reproduce the bug on my end. If I set sufix to none, I can see the images by clicking the VideoWriter while nothing is saved. If I set it to Filecount, the images are written down. Is there anyway to debug on my side?

**Reply by Andrianarivelo** [reply]
I have the same did you find any solution to this ?

**Reply by bruno-f-cruz** [reply]
Can you share the workflow?
